generic/tclsass.c

```diff
@@ -1420,7 +1422,12 @@ static int SassObjCmd(
 
 done:
     if (optsPtr != NULL) {
-	free(optsPtr); /* HACK: No official destructor function. */
```

Comment:
It might be nice to retain this comment (or something like it) in the "#else" block.